src/contrib/testkits/Akka.TestKit.Xunit.Tests/Akka.TestKit.Xunit.Tests.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="../../../xunitSettings.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkTestVersion);net8.0</TargetFrameworks>
```

Comment:
Well, that's going to be a bit of a breaking change but you know, at some point we have to step into the future here.